torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -2026,6 +2033,27 @@ void ONNXShapeTypeInference(
   }
 
   SpecialPostProcess(n);
+  // Get data propagation result from ONNX shape inference
+  for (auto output: n->outputs()) {
+    if (generated_shape_data.count(outputs_map[output->debugName()]) > 0) {
+      auto shape_data = generated_shape_data.find(outputs_map[output->debugName()])->second;
```

Comment:
Avoid looking this up repeatedly in the map by saving an iterator:
save the result of `find`. Something like:
```c++
auto generated_shape_pair = generated_shape_data.find(outputs_map[output->debugName()]);
if (generated_shape_pair != generated_shape_data.end()) {
   auto& generated_shape = generated_shape_pair->second;
   ...
   generated_shape_data.erase(generated_shape_pair);
```